docs/how-to-guides/monitoring-with-prometheus-operator.md

```diff
@@ -7,12 +7,12 @@
 * [Deploy Prometheus Operator](#deploy-prometheus-operator)
   * [Configure Prometheus Operator](#configure-prometheus-operator)
   * [Install Prometheus Operator](#install-prometheus-operator)
-* [Accessing Prometheus, Alertmanager and Grafana](#accessing-prometheus-operator-sub-components)
-  * [Accessing Prometheus](#accessing-prometheus)
+* [Access Prometheus, Alertmanager and Grafana](#access-prometheus-alertmanager-and-grafana)
+  * [Access Prometheus](#access-prometheus)
     * [Using port forward](#using-port-forward)
     * [Using Ingress](#using-ingress)
-  * [Accessing Alertmanager](#accessing-alertmanager)
-  * [Accessing Grafana](#accessing-grafana)
+  * [Access Alertmanager](#access-alertmanager)
+  * [Access Grafana](#access-grafana)
     * [Using port forward](#using-port-forward-1)
```

Comment:
Still `Using` here :smile: it seems present continuous would be better after all? :smile:

Reply:
Oh I kept it like that because it does not sound right when we say `Access Grafana Use port forward`, but `Access Grafana using port forward`. It is like a complete sentence.

Reply:
Hm true, that make sense. But I'm not 100% convinced, so maybe someone else can have a look as well :)